scheduler/src/cook/kubernetes/api.clj

```diff
@@ -1068,6 +1079,7 @@
 
     (.addTolerationsItem pod-spec toleration-for-deletion-candidate-of-autoscaler)
     (.addTolerationsItem pod-spec (toleration-for-pool cook-pool-taint-name cook-pool-taint-prefix pool-name))
+    (.addTolerationsItem pod-spec toleration-tenured-node)
```

Comment:
Won't this make synthetic pods tolerate the taint?

Reply:
.... yes, they would. Good catch. Thanks.